pkg/skaffold/kubernetes/portforward/resource_forwarder.go

```diff
@@ -66,12 +66,11 @@ func (p *ResourceForwarder) Start(ctx context.Context) error {
 
 // Port forward each resource individually in a goroutine
 func (p *ResourceForwarder) portForwardResources(ctx context.Context, resources []*latest.PortForwardResource) {
-	for _, r := range resources {
-		r := r
-		go func() {
+	go func() {
```

Comment:
Not sure, if we are slowing down skaffold portforward phase due to sequentially forwarding resources.

Other alternative would be at L63, we first port forward `uderDefinedResources` in parallel. Wait for it to complete and then portforward serviceResources.

I don't mind this approach either.

Reply:
Thanks for the idea @tejal29. You know what, let's go with correctness and if we have an issue, we'll improve the perf.